### PR TITLE
ref(transcriptions): refactor transcriptions api 

### PR DESCRIPTION
**Pull request by quitrk**

### CODE REVIEW

File: react/features/subtitles/middleware.ts [hunk 1]
```diff
@@ -157,9 +159,13 @@ function _endpointMessageReceived({ dispatch, getState }: IStore, next: Function
 
             // Notify the external API too.
             if (typeof APP !== 'undefined') {
+                const sanitizedTranscriptMessage = { ...newTranscriptMessage };
```
Comment (saghul) [inline react/features/subtitles/middleware.ts:162]: Let's please modify this to match the newly added webhook. Sample payload:

```
      "messageID": "018c1abc-de07-77cf-b73e-147bf6340b1b",
      "final": " This",
      "language": "en",
      "participant": {
        "avatarUrl": "https://www.gravatar.com/avatar/583ddb012250f167d239434148feecc0?d=wavatar&size=200",
        "name": "mihdmt",
        "id": "asdfg123"
      }
```

All the necessary fields that are missing should be present in the payload we get as an endpoint message.

**Review by saghul — COMMENTED** [review]
Looking good! One request to change the iframe event payload and we're good to go.

File: react/features/subtitles/middleware.ts (hunk 1 above)
Comment (quitrk) [reply react/features/subtitles/middleware.ts:162]: I knew I forgot something !!!

**Review by saghul — APPROVED** [review]
Excellent!

**Pull request merged**